### PR TITLE
add Gaya to the README

### PR DESCRIPTION
**Pull request by gayapedersen**
fixes issue #3

### CODE REVIEW

**Comment by sampoornaravikanth** [comment]
Im leaving a comment

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# 2017-Training
\ No newline at end of file
+# 2017-Training
+
```
Comment (porglezomp) [inline README.md:2]: This blank line is very blank.

**Comment by a2aaron** [comment]
comment is here

File: README.md [hunk 2]
```diff
@@ -1 +1,3 @@
-# 2017-Training
\ No newline at end of file
+# 2017-Training
```
Comment (porglezomp) [inline README.md:1]: The title looks good!

**Review by porglezomp — APPROVED** [review]
This looks good to me!

**Review by porglezomp — APPROVED** [review]
Looks good to me

**Pull request merged**